src/status_im2/contexts/wallet/account/tabs/view.cljs

```diff
@@ -1,13 +1,57 @@
 (ns status-im2.contexts.wallet.account.tabs.view
   (:require
     [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
     [react-native.core :as rn]
     [status-im2.contexts.wallet.account.style :as style]
     [status-im2.contexts.wallet.common.temp :as temp]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn description
+  []
+  [quo/text {:size :paragraph-2}
+   (map (fn [network]
+          ^{:key (str network)}
+          [quo/text
+           {:size   :paragraph-2
+            :weight :medium
+            :style  {:color (get colors/networks network)}}
+           (str (subs (name network) 0 3) (when (= network :arbitrum) "1") ":")])
+        temp/network-names)
```

Comment:
do we not have these networks somewhere more permanent? or this would come from the backend? 
cc @briansztamfater @smohamedjavid

Reply:
@J-Son89 I think we will find out when we starting working more on the backend

Reply:
We will for sure need to define this somewhere, I am noticing we are starting to repeat this kind of code. Probably the best would be to adopt whatever approach was implemented in `status-desktop`

Reply:
I can move them to `status-im2.constants` maybe. Right now there is actually one definition of `(def network-names [:ethereum :optimism :arbitrum])` in this `temp` file